conf/machine/khadas-vim3.conf

```diff
@@ -3,9 +3,18 @@
 require conf/machine/include/amlogic-a311d.inc
 require conf/machine/include/khadas-vim3-dtb.inc
 
-KERNEL_IMAGETYPE = "Image"
-IMAGE_BOOT_FILES_remove = "uImage"
-IMAGE_BOOT_FILES_append = " Image"
+#
+# Kernel
+# - Use standard linux-yocto recipe, drop custom uImage, build FIT image
+#
+PREFERRED_PROVIDER_virtual/kernel_meson-gx ?= "linux-yocto-dev"
+LINUX_VERSION_meson-gx = "5.8"
```

Comment:
Is this mandatory to be fixed ?

Reply:
hmm, no.  Good point.  In fact, with linux-yocto-dev, we should let it pick the latest version.  I think it follows mainline -rc releases.  Also, that recipe has `KERNEL_VERSION_SANITY_SKIP = "1"` so it won't warn anyways if there's a version mismatch.

Note that I put linux-yocto-dev so we could havea v5.8+ kernel without a pile of patches.  When linux-yocto updates to v5.8, we could switch to that to not have a moving kernel version.

Reply:
Pushed an updated version without a fixed kernel version.